### PR TITLE
[Crane] Replacing AnimatedContent with Pager, and LazyGrid with LazyStaggeredGrid

### PR DESCRIPTION
**Pull request by riggaroo**
Replacing `AnimatedContent` with [`HorizontalPager`](https://developer.android.com/jetpack/compose/layouts/pager). There is nothing incorrect about using `AnimatedContent`, the contents of my earlier [blog post](https://medium.com/androiddevelopers/customizing-animatedcontent-in-jetpack-compose-629c67b45894) are still valid. However, in this use case, a `Pager` is more correct for the UX of this screen, users expect to be able to swipe between tabs. 

https://user-images.githubusercontent.com/9973046/235145534-7fa93dc1-e1fa-428b-88fe-3523dc220690.mp4

Replacing `LazyGrid` with `LazyStaggeredGrid`
Screenshot_20230428_131844

### CODE REVIEW

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt [hunk 1]
```diff
@@ -98,15 +89,15 @@ fun ExploreSection(
                             }
                         }
                     }
-                    item(span = {
+                    /*item(span = {
```
Comment (JolandaVerhoef) [inline Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt:92]: What's this? If you're commenting this out, shouldn't you just remove it?

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt [hunk 2]
```diff
@@ -182,39 +173,23 @@ private fun ExploreItemRow(
...
 private fun ExploreImage(item: ExploreModel) {
-    Box {
-        val painter = rememberAsyncImagePainter(
-            model = ImageRequest.Builder(LocalContext.current)
-                .data(item.imageUrl)
-                .crossfade(true)
-                .build()
-        )
-
-        if (painter.state is Loading) {
```
Comment (JolandaVerhoef) [inline Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt:193]: Why did you remove this loading state?

File: Crane/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
+#Fri Apr 28 12:09:33 SAST 2023
```
Comment (JolandaVerhoef) [inline Crane/gradle/wrapper/gradle-wrapper.properties:1]: remove

File: Crane/gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,6 @@
...
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```
Comment (JolandaVerhoef) [inline Crane/gradle/wrapper/gradle-wrapper.properties:4]: Why did you downgrade this? Does AS stable not work with Gradle 8.1?

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt (hunk 1 above)
Comment (riggaroo) [reply Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt]: Forgot about it - updated now to include it again.

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt (hunk 2 above)
Comment (riggaroo) [reply Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt:193]: It didn't look good when switching to different sized images, having nothing instead looks better than the jump in size change.

**Review by JolandaVerhoef — APPROVED** [review]
